### PR TITLE
AC: fix padding if resize enabled

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py [hunk 1]
```diff
@@ -271,9 +272,10 @@ def process(self, image, annotation_meta=None):
             'height': height,
             'resized': False
         }
-        if image.data.shape[:2] != (pref_height, pref_width):
-            image.data = cv2.resize(image.data, (pref_height, pref_width))
+        if self.enable_resize and image.data[:2] != [pref_height, width_pref_init]:
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py:275]:
```suggestion
        if self.enable_resize and image.data.shape[:2] != [pref_height, width_pref_init]:
```

**Review by Anna-Mironova — APPROVED** [review]

**Comment by eaidova** [comment]
Jenkins please retry a build

**Comment by eaidova** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/accuracy_checker/launcher/tf_launcher.py [hunk 2]
```diff
@@ -299,6 +299,21 @@ def _load_saved_model(self, model_dir):
 
         return graph
 
+    def fit_to_input(self, data, layer_name, layout, precision):
+        layer_shape = self.inputs[layer_name]
+        if (
+            len(layer_shape) > len(np.shape(data)) and
+            len(np.squeeze(np.zeros(layer_shape))) == len(np.squeeze(np.zeros(np.shape(data))))
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/launcher/tf_launcher.py:306]:
```suggestion
                len(layer_shape) > len(np.shape(data)) and
                len(np.squeeze(np.zeros(layer_shape))) == len(np.squeeze(np.zeros(np.shape(data))))
```

**Pull request merged**